src/assets/HTMLAsset.js

```diff
@@ -19,10 +19,9 @@ const ATTRS = {
     'iframe',
     'embed'
   ],
-  href: ['link', 'a', 'use'],
+  href: ['link', 'a'],
   srcset: ['img', 'source'],
   poster: ['video'],
-  'xlink:href': ['use'], // Deprecated since SVG 2, throws error in svgo
```

Comment:
FYI Chrome doesn’t support the plain `href` so it can be “deprecated” but support is 0 as far as I’m concerned. Does this change still work?

Reply:
It's still supported in this PR, don't wanna break cross-browser support.
Dependency collection is just moved from `collectDependencies` into `svgDependencyWalker` in this pr (as htmlnano stringifies svgs...)